src/util/mod.rs

```diff
@@ -1,13 +1,18 @@
-use std::{env, sync::Once};
+use std::{env, str::FromStr, sync::Once};
 
 use electrum_client::ElectrumApi;
-use elements::encode::Decodable;
+use elements::{encode::Decodable, hex::ToHex};
+use lightning_invoice::{Bolt11Invoice, RouteHintHop};
 
 use crate::{error::Error, network::electrum::ElectrumConfig};
 
 pub mod ec;
 pub mod secrets;
 
+const ENDPOINT: &str = "https://api.testnet.boltz.exchange";
```

Comment:
Not a fan of hardcoding that here. I don't think it is being used

Reply:
Removed..